loops/common-lisp/code.lisp

```diff
@@ -0,0 +1,26 @@
+#-sbcl (error "only Steel Bank Common Lisp supported, sorry")
+
+(defun main ()
+  (declare (optimize (speed 3) (safety 0)))
+  (let ((r (random 10000 (make-random-state t)))
+        (a (make-array 10000   :element-type '(unsigned-byte 32) ))
+        (u (parse-integer (second *posix-argv*)) ))
+    (declare
+     (type (mod 10000) r)
+     (type (unsigned-byte 32) u)
+     (type (simple-array (unsigned-byte 32) (10000)) a))
+
+
+    (loop for i of-type (mod 10001) from 0 below 10000 do
+      (loop for j of-type (mod 10001) from 0 below 10000 do
```

Comment:
shouldn't the [inner loop be 100k](https://github.com/bddicken/languages/blob/7d2d3ee6691335456a499975cfb2b1e9fe4937da/loops/c/code.c#L10), not 10k?🤔

Reply:
quite so. A dazzling feat of 0s.